tox.ini

```diff
@@ -1,4 +1,9 @@
+[flake8]
+exclude = .idea,.git,venv*/,.eggs/,*.egg-info
+max-line-length = 100
+show-source = True
+
 [pytest]
 DJANGO_SETTINGS_MODULE = tests.dj.settings
-addopts = --random-order --cov-report=xml --cov=./dj_cqrs
+addopts = --cov-report=xml --cov=./dj_cqrs
```

Comment:
Why test randomization is off?

Reply:
It's still on, another plugin is used `pytest-randomly`